Add simulation data to unitGroup.

youngca opened the pull request.

youngca committed:
Add simulation data to unitGroup.

eric79 commented on support/proxy/vwf.example.com/mil-sym/unitSimData.vwf.yaml:
```diff
@@ -0,0 +1,37 @@
+      }
+    value: false
+
+events:
```
No need to include `events`, `methods`, or `scripts` if they are not used.  :smile:

eric79 commented:
:+1:

youngca committed:
When a user starts dragging a unit group, pop it up to the top (local view only).

youngca committed:
Set defaults for unit symbol fill colors to be consistent with the spec.

Despite the fact that mil-sym is developed by the Army, the unit symbol fill colors were using the
spec's color for unfilled symbols, not filled symbols.

youngca committed:
Remove unused events, methods and scriprs.

eric79 commented:
:+1:

youngca committed:
Post-code review cleanup.  Use dot notation and add TODO comment.
